src/templates/laravel/LaravelStack.js

```diff
@@ -48,7 +50,13 @@ export default class LaravelStack extends Stack {
     });
     this.ejectables.set('apache2-site.conf', {
       label: 'Apache virtual host',
-      path: '/etc/apache2/site-available/000-default.conf',
+      path: '/etc/apache2/site-available/app.conf',
+      service: 'laravel',
+    });
+
+    this.runnables.set('apache-reload', {
+      label: 'Reload Apache service',
+      exec: 'service apache2 reload',
       service: 'laravel',
     });
```

Comment:
Another runnable we could have on `laravel` is `phpunit`. A command that will run the tests.

Not sure if `phpunit` is available inside our container image. If not, let's queue this up in Trello and handle it later.